semgrep/semgrep/config_resolver.py

```diff
@@ -61,26 +64,165 @@
 }
 
 
+class ConfigPath:
+    def __init__(self, config_str: str) -> None:
+        """ returns the registry url for the config_str, if it is a registry entry """
+        if config_str in RULES_REGISTRY:
+            self.from_registry = True
```

Comment:
@emjin Make these private and give them default values

Reply:
+1